apps/vast/pyvast_threatbus/pyvast_threatbus.py

```diff
@@ -370,6 +378,7 @@ async def retro_match_vast(
         logger.debug(f"Retro-matched {reported} sighting(s) for indicator: {indicator}")
         s_retro_matches_per_ioc.observe(reported)
         s_retro_query_time_s_per_ioc.observe(time.time() - start)
+        g_retro_match_backlog.dec()
```

Comment:
Isn't missing decrements when we return from this function with an error or an early return?

Reply:
Good point, I updated the missing places.